### ISSUE TITLE
Disconnected from game?

### ISSUE THREAD

**Issue by xvento** [GlowstoneMC/Glowstone-Legacy#643]
Hello, I need some help.. 
The server is up and running but everytime i try entering it the code: 
![image](https://cloud.githubusercontent.com/assets/12885315/8149967/c4216926-12d7-11e5-831d-59c4516865ec.png)
 What can I do? I'm not that hightech, please help me. :)

**Comment by greatman** [comment]
Is there any issues (stacktraces) on the server side?

**Comment by xvento** [comment]
No, not what I know of... There are three people who can't get in and get this exact message, we live in the same claimed chunk :/ ... 
There are 10 others online and they don't have the problem..

**Comment by gdude2002** [comment]
When do you get this error? On join? On going to a certain part of the map?

EDIT: I just reread, saw it was on join. Are you using offline mode?

**Comment by xvento** [comment]
On a certaint point of a map :)
No I'm not using offline :) 

This error comes on when I join the server

**Comment by coelho** [comment]
@xvento Please paste the console output.
